nixos/modules/security/setuid-swap.c

```diff
@@ -0,0 +1,36 @@
+#define _GNU_SOURCE
```

Comment:
No, I meant for the whole file, would be nice to have a description what these 36 lines do :)